nixos/modules/services/networking/alice-lg.nix

```diff
@@ -0,0 +1,108 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.alice-lg;
+  settingsFormat = pkgs.formats.ini { };
+in
+{
+  options = {
+    services.alice-lg = {
+      package = mkOption {
+        type = types.package;
+        default = pkgs.alice-lg;
+        defaultText = literalExpression "pkgs.alice-lg";
+        description = lib.mdDoc "The Alice Looking Glass package to use.";
+      };
+      enable = mkEnableOption (lib.mdDoc "Alice Looking Glass");
+
+      settings = mkOption {
+        type = settingsFormat.type;
+        default = { };
+        description = lib.mdDoc ''
+          System-wide configuration for GameMode (/etc/gamemode.ini).
```

Comment:
This seems irrelevant?

Reply:
Yes, I removed it, it was from the module I initially copied.